cmds/modules/noded/main.go

```diff
@@ -193,6 +195,27 @@ func action(cli *cli.Context) error {
 		return hypervisor, nil
 	})
 
+	log.Info().Msg("start perf scheduler")
+	perfMon, err := perf.NewPerformanceMonitor("unix://var/run/redis.sock")
+	if err != nil {
+		return errors.Wrap(err, "failed to create a new perfMon")
+	}
+	defer perfMon.Close()
+
+	err = perfMon.Run(ctx)
+	if err != nil {
+		log.Error().Err(err).Msg("failed to run the scheduler")
+	}
+	bus.WithHandler("zos.perf.get", func(ctx context.Context, payload []byte) (interface{}, error) {
+		var taskName string
+		err := json.Unmarshal(payload, &taskName)
+		if err != nil {
+			return nil, errors.Wrapf(err, "failed to unmarshal payload: %v", payload)
+		}
+
+		return perfMon.GetCache(taskName)
```

Comment:
I think the API need to support returning all available tests, or specific one by name. so this function need to support not having a `test-name` and that name it call be results from all. 

Also the `GetCache` is not a good name. Again this exposes the details of an implementation. The user of the API does not care where the result is stored or if the test is executed on request. Hence this should only called `Get`